### ISSUE TITLE
send() output broken in 2.1.0

### ISSUE THREAD

**Issue by Tasssadar** [frida/frida-tools#23]
Hello, PR #8 breaks the send api in Frida scripts. Put `send({ "foo": "bar"});` or `send('{ "foo": "bar" }');` to a Frida script, run it with repl and you get this exception:

```Exception in thread Thread-1:
Traceback (most recent call last):
  File "threading.py", line 916, in _bootstrap_inner
  File "threading.py", line 864, in run
  File "frida_tools/application.py", line 452, in _run
  File "frida_tools/repl.py", line 158, in <lambda>
  File "frida_tools/repl.py", line 444, in _process_message
  File "frida_tools/application.py", line 351, in _print
TypeError: sequence item 1: expected str instance, dict found
```
I tried to fix it and submit PR, but the [logic added in #8](https://github.com/frida/frida-tools/blob/d321d46755d11ae3422ce20ae9d5752cf4de900f/frida_tools/application.py#L351) completely bypasses the code in `_print` that's designed to handle non-string args by joining all args to a single string.

If I'm reading it correctly, it also changes behavior of  `_print()` with multiple args:

- Before PR: `self._print("a", "b")` -> `a b`
- After PR: `self._print("a", "b")` -> `ab`

So I'm not sure what you want the fix to be, which behavior of `_print` is desired.

**Comment by TheDauntless** [comment]
This is indeed broken. Basic test case on Android:

```javascript
//hook.js
Java.perform(myFunction);

function myFunction()
{
send("Hello world");
}

//command
frida -U -f com.tripadvisor.tripadvisor -l hook.js --no-pause
```

Output:

```Traceback (most recent call last):
  File "/usr/local/Cellar/python/3.6.5/Frameworks/Python.framework/Versions/3.6/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/local/Cellar/python/3.6.5/Frameworks/Python.framework/Versions/3.6/lib/python3.6/threading.py", line 864, in run
    self._target(*self._args, **self._kwargs)
  File ".../env/lib/python3.6/site-packages/frida_tools/application.py", line 452, in _run                  
    work()
  File ".../env/lib/python3.6/site-packages/frida_tools/repl.py", line 158, in <lambda>
    self._reactor.schedule(lambda: self._process_message(message, data))
  File ".../env/lib/python3.6/site-packages/frida_tools/repl.py", line 444, in _process_message
    self._print("message:", message, "data:", data)
  File ".../env/lib/python3.6/site-packages/frida_tools/application.py", line 351, in _print
    args = "".join(arg for arg in args if not self._plain_terminal or not arg.startswith("\033"))
TypeError: sequence item 1: expected str instance, dict found
```

Temporary fix is of course to downgrade to 2.0.2:

`pip install frida-tools==2.0.2`

**Comment by oleavr** [comment]
Fixed in `2.1.1`.
